strings/group_anagrams.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+import doctest
```

Comment:
Let's move this line down under `if __name__ == "__main__":` so it is not imported if someone uses this file as a module.